near-sdk/src/environment/hash.rs

```diff
@@ -6,27 +8,36 @@ mod private {
 
     impl Sealed for super::Sha256 {}
     impl Sealed for super::Keccak256 {}
+    impl Sealed for super::Identity {}
 }
 
-/// Cryptographic hashes that can be used within the SDK as a hashing function.
-pub trait CryptoHasher: self::private::Sealed {
-    /// Output type of the hashing function.
-    type Digest;
+/// Trait used to generate keys to store data based on a serializable structure.
+pub trait StorageKeyer: self::private::Sealed {
```

Comment:
Usually traits are named as verbs, so perhaps something like?

```rust
trait ToKey {
  fn to_key()
}
```

Reply:
Yeah, this naming is probably better, I'll switch